plugins/sqs/sqs.pc

```diff
@@ -0,0 +1,11 @@
+prefix=/usr
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+includedir=${prefix}/include
+
+Name: SQSHPCC
+Description: This is a plugin for HPCC allowing to dialog with Amazon's SQS 
```

Comment:
@richardkchapman sorry to bother but I was logged in incorrectly for this comment. I use lexisdehilster and somehow had another login. Can you delete this?